index.js

```diff
@@ -289,6 +289,16 @@ class YouTubePlayer extends EventEmitter {
           loadCb(null, window.YT)
         }
       }
+    } else {
+      // if a youtube iframe ready event handler is already declared,
+      // don't overwrite it. Declare our own and forward the ready call.
+      const previous = window.onYouTubeIframeAPIReady
+      window.onYouTubeIframeAPIReady = () => {
+        if (previous) {
+           previous()
+        }
```

Comment:
Needs a
`
const loadCb = loadIframeAPICallbacks.shift();
`

Reply:
I'm not sure I understand what you mean 🤔

Reply:
Sure! Try running your patch/PR -- you'll see that when **loadCb** (line 300) hits, **loadCb** is undefined. Adding `const loadCb = loadIframeAPICallbacks.shift();` before `loadCb(null, window.YT)` fixes it.